src/Microsoft.AspNet.Razor/SourceLocation.cs

```diff
@@ -112,33 +164,43 @@ private static SourceLocation CreateUndefined()
             return sl;
         }
 
-        public static bool operator <(SourceLocation left, SourceLocation right)
```

Comment:
I'm not sure how attached we are to these operators - they are scarcely used in our code.